test/csuite/schema_abort/main.c

```diff
@@ -849,6 +888,15 @@ run_workload(uint32_t nth)
      */
     testutil_check(session->close(session, NULL));
 
+    if (opts->tiered_storage) {
+
```

Comment:
Remove this blank line

Reply:
Fixed in f2a9620ccbcd7387228e9f631319df147093c77c